src/conduits/RwaInputConduit3.sol

```diff
@@ -1,6 +1,9 @@
 // SPDX-FileCopyrightText: © 2022 Dai Foundation <www.daifoundation.org>
 // SPDX-License-Identifier: AGPL-3.0-or-later
 //
+// Copyright (C) 2020-2021 Lev Livnev <lev@liv.nev.org.uk>
+// Copyright (C) 2021-2022 Dai Foundation
+//
```

Comment:
Even if we're keeping Lev, we should use the new format:

```suggestion
// SPDX-FileCopyrightText: © 2020-2021 Lev Livnev <lev@liv.nev.org.uk>
// SPDX-FileCopyrightText: © 2022 Dai Foundation <www.daifoundation.org>
// SPDX-License-Identifier: AGPL-3.0-or-later
//
```

Reply:
done